src/test/java/com/woowacamp/soolsool/integration/repository/LiquorRegionRepositoryTest.java

```diff
@@ -20,7 +21,7 @@ class LiquorRegionRepositoryTest {
     @DisplayName("LiquorRegionStatus의 name를 가지고 LiquorRegion을 조회한다.")
     void findByLiquorRegionType_type() {
         // given
-        LiquorRegion 경기도 = liquorRegionRepository.findByType(GYEONGGI_DO);
+        LiquorRegion 경기도 = liquorRegionRepository.findByType(GYEONGGI_DO).orElseThrow();
```

Comment:
.orElseThrow를 다음 줄로 넘겼으면 좋겠어요!!!
```suggestion
        LiquorRegion 경기도 = liquorRegionRepository.findByType(GYEONGGI_DO)
            .orElseThrow();
```